src/inference/src/ie_system_conf.cpp

```diff
@@ -167,6 +191,224 @@ int getNumberOfLogicalCPUCores(bool bigCoresOnly) {
 #    endif
     return logical_cores;
 }
+
+struct CPU {
+    int _processors = 0;
+    int _sockets = 0;
+    int _cores = 0;
+
+    std::vector<std::vector<int>> _proc_type_table;
+    std::vector<std::vector<int>> _cpu_mapping_table;
+
+    CPU() {
+#    ifdef _WIN32
```

Comment:
Let's still keep system information collection in `lin_system_conf.cpp` and `win_system_conf.cpp`.

Reply:
Ok, done.